test_dll.py

```diff
@@ -0,0 +1,185 @@
+from __future__ import unicode_literals
+import pytest
+from dll import Node
+from dll import DLL
+
+
+def test_node_init():
+    ''' Test node init. '''
+    m = Node(3)
+    assert m.val == 3
+    assert m.behind is None
+
+
+def test_dll_init():
+    ''' Test dll init. '''
+    l = DLL()
+    assert l.head is None
+    assert l.tail is None
+
+
+def test_dll_append():
+    ''' Test append method. '''
```

Comment:
Same story here as on the queue - too many asserts per test, so you won't get useful enough feedback from py.test. Try building your DLL in a fixture and testing it with the functions.